### PR TITLE
Document for api delete comments

### PR DESCRIPTION
**Pull request by at-minhdao**
## Description
- Document for api delete comments
## Reference Requirements
- Product Backlog ID: PBI-None
- Task ID: TI-None

## Reference Isuees
#{issue_id}

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)
- Non-SQL
- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

**Review by at-trampham — COMMENTED** [review]
LGTM

File: docs/api-references/comment.md [hunk 1]
```diff
@@ -204,3 +204,33 @@ Create new Comment
...
+| status | String | Status result |
+| code | Number | HTTP status code |
+```json
+{
+    "meta": {
+        "status": "Successfully",
+        "code": 200
+    },
+}
+```
```
Comment (at-duyho) [inline docs/api-references/comment.md]: - trường hợp fail đâu?

**Review by ghost — CHANGES_REQUESTED** [review]
check my comment

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**